src/drivers/heater/heater.h

```diff
@@ -163,9 +164,32 @@ class Heater : public ModuleBase<Heater>, public ModuleParams, public px4::Sched
 	 */
 	void update_params(const bool force = false);
 
+	/**
+	 * @brief Enables / configures the heater (either by GPIO or PX4IO)
+	 */
+	void heater_enable(void);
+
+	/**
+	 * @brief Disnables the heater (either by GPIO or PX4IO)
+	 */
+	void heater_disable(void);
+
+	/**
+	 * @brief Turns the heater on (either by GPIO or PX4IO)
+	 */
+	void heater_on(void);
+
+	/**
+	 * @brief Turns the heater off (either by GPIO or PX4IO)
+	 */
+	void heater_off(void);
+
 	/** Work queue struct for the RTOS scheduler. */
 	static struct work_s _work;
```

Comment:
This reminds me I had a cleanup PR moving this to the new PX4 work queue and added some uORB logging.